File: lab01/src/problem_6_3/Triangle.java
package problem_6_3;
import java.util.*; 
class Triangle { 
	public static void main(String[] args) 
	{ 
		Scanner sc = new Scanner(System.in); 
		System.out.print("Input the number of rows to be printed: "); 
		int n = sc.nextInt(); 

		for (int i = 1; i <= n; i++) { 
			for (int j = n; j >= i; j--) { 
				System.out.print(" "); 
			} 
			for (int j = 1; j <= i; j++) { 
				System.out.print("* "); 
			} 
			System.out.println(); 
		} 
        sc.close();
	} 
} 

File: lab01/src/com/oop/problem6_3/Triangle.kt
package com.oop.problem6_3

fun main() {

    print("Input the number of rows to be printed: ")
    val n: Int = readLine()!!.toInt()
    for (i in 1..n) {
        for (j in n downTo i) {
            print(" ")
        }
        for (j in 1..i) {
            print("* ")
        }
        println()
    }
}